src/azure-cli/azure/cli/command_modules/acr/_stream_utils.py

```diff
@@ -191,6 +202,24 @@ def _stream_logs(no_format,  # pylint: disable=too-many-locals, too-many-stateme
             raise CLIError("Run was canceled")
 
 
+def _stream_artifact_logs(log_file_sas,  # pylint: disable=too-many-locals, too-many-statements, too-many-branches
```

Comment:
does the following code trigger the lint warnings?

Reply:
i dont think so. will disable it.